CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
To sendo chato já hahah mas acho que isso é `added`, é uma nova feature, nao um bug fix ao meu ver

Reply:
Eu fui nessa linha de pensamento quanto à classificação. O que estava lá antes era inesperado.

Reply:
ok, faz sentido